--- a/src/gpu/initialize_gpu.cu
+++ b/src/gpu/initialize_gpu.cu
@@ -241,11 +241,16 @@
       }else{
         fprintf(fp,"  canMapHostMemory: FALSE\n");
       }
+#if CUDA_VERSION < 13000 || (defined (__CUDACC_VER_MAJOR__) && (__CUDACC_VER_MAJOR__ < 13))
       if (deviceProp.deviceOverlap){
         fprintf(fp,"  deviceOverlap: TRUE\n");
       }else{
         fprintf(fp,"  deviceOverlap: FALSE\n");
       }
+#else
+      // CUDA version >= 13, deviceOverlap deprecated, replaced by asyncEngineCount
+      fprintf(fp,"  asyncEngineCount: %d\n", deviceProp.asyncEngineCount);
+#endif
       if (deviceProp.concurrentKernels){
         fprintf(fp,"  concurrentKernels: TRUE\n");
       }else{
